xwayland.c

```diff
@@ -165,6 +174,8 @@ handle_xwayland_surface_new(struct wl_listener *listener, void *data)
 
 	view_init(&xwayland_view->view, server, CAGE_XWAYLAND_VIEW, &xwayland_view_impl);
 	xwayland_view->xwayland_surface = xwayland_surface;
+	xwayland_view->view.lx = xwayland_surface->x;
+	xwayland_view->view.ly = xwayland_surface->y;
```

Comment:
Hm, should we add a `if (!xwayland_view_should_manage(view))` here?

Reply:
Then it wouldn't cover the case where override_redirect is set between creating and mapping the window, e.g.:

```c
Window w = XCreateSimpleWindow(dpy, root, 100, 100, 50, 50, 3, 0xffff00, 0xff00ff);

XSetWindowAttributes attrs = { .override_redirect = True };
XChangeWindowAttributes(dpy, w, CWOverrideRedirect, &attrs);

// Should appear at position (100, 100)
XMapWindow(dpy, w);
XFlush(dpy);
```

Reply:
Hm… What about override redirect changing after the window is unmapped? Sounds like we should probably set up a `set_override_redirect` listener to handle all cases correctly?

I'd be more comfortable only copying the X11 position when the window is O-R.